source/javascripts/components/common/notifications/CreatingYmlOnWebsiteProgress.tsx

```diff
@@ -0,0 +1,8 @@
+import React, { FC } from "react";
+import { Notification } from "@bitrise/bitkit";
+
+const CreatingYmlOnWebsiteProgress: FC = () => (
+	<Notification type="progress">Creating bitrise.yml on bitrise.io...</Notification>
```

Comment:
Can we avoid these duplications? Seems like only the text is different in these components.

Reply:
Well the reason behind creating these components was to avoid having the same text in multiple places (in case we have to change the wording) and to make the React components using them a bit easier to read (most cases these notifications are used in ternary operators, for me it's easier to understand them if it's just one simple tag).